CONTRIBUTING.rst

```diff
@@ -0,0 +1,114 @@
+.. highlight:: shell
+
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/sul-cidr/histonets-cv/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+and "help wanted" is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "enhancement"
+and "help wanted" is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+Histonets Computer Vision could always use more documentation, whether as part of the
+official Histonets Computer Vision docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at https://github.com/sul-cidr/histonets-cv/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `histonets-cv` for local development.
+
+1. Fork the `histonets-cv` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/histonets-cv.git
+
+3. Install your local copy into a virtualenv. Assuming you have virtualenvwrapper installed, this is how you set up your fork for local development::
+
+    $ mkvirtualenv histonets
+    $ cd histonets/
+    $ python setup.py develop
```

Comment:
Yep, OpenCV might be a problem since it is a bit cumbersome to install. The package `opencv-python` provides compiled binaries with limited options for a few Python versions and architectures. In [its PyPI page](https://pypi.python.org/pypi/opencv-python/3.1.0.3) you can find which ones are available. I am using Python 3.5.2. Other good instructions to get OpenCV working in a virtualenv:
- On Ubuntu: http://www.pyimagesearch.com/2015/07/20/install-opencv-3-0-and-python-3-4-on-ubuntu/
- On Mac OSX: http://www.pyimagesearch.com/2015/06/29/install-opencv-3-0-and-python-3-4-on-osx/